deps.bzl

```diff
@@ -6991,3 +6986,104 @@ def install_static_dependencies(workspace_name = "buildbuddy"):
+        ]),
+        urls = ["https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v2.1.22/docker-credential-gcr_linux_amd64-2.1.22.tar.gz"],
+        sha256 = "443e897dc383d69e55e6dbcb13802f4ec88444848612e83f0381df2ddd721694",
+    )
+    http_archive(
+        name = "com_github_googlecloudplatform_docker_credential_gcr-linux-arm64",
+        build_file_content = "\n".join([
+            'package(default_visibility = ["//visibility:public"])',
+            'filegroup(name = "docker-credential-gcr.bin", srcs = ["docker-credential-gcr"])',
+        ]),
+        urls = ["https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v2.1.22/docker-credential-gcr_linux_arm64-2.1.22.tar.gz"],
+        sha256 = "b607dfb7516dc1ca6a2a05322d938cea58cc5975e2063efc3695ee6ddb2dccc7",
+    )
+
+    http_archive(
+        name = "com_github_rootless_containers_rootlesskit-linux-amd64",
+        build_file_content = "\n".join([
+            'package(default_visibility = ["//visibility:public"])',
+            'filegroup(name = "rootlessctl.bin", srcs = ["rootlessctl"])',
+            'filegroup(name = "rootlesskit.bin", srcs = ["rootlesskit"])',
+            'filegroup(name = "rootlesskit-docker-proxy.bin", srcs = ["rootlesskit-docker-proxy"])',
+        ]),
+        urls = ["https://github.com/rootless-containers/rootlesskit/releases/download/v2.1.0/rootlesskit-x86_64.tar.gz"],
+        sha256 = "1e4609a3d0ca67c4fc99fe9121870cf5ca46a438f43b151e9b54456d166f4b10",
+    )
+    http_archive(
+        name = "com_github_rootless_containers_rootlesskit-linux-arm64",
+        build_file_content = "\n".join([
+            'package(default_visibility = ["//visibility:public"])',
+            'filegroup(name = "rootlessctl.bin", srcs = ["rootlessctl"])',
+            'filegroup(name = "rootlesskit.bin", srcs = ["rootlesskit"])',
+            'filegroup(name = "rootlesskit-docker-proxy.bin", srcs = ["rootlesskit-docker-proxy"])',
+        ]),
+        urls = ["https://github.com/rootless-containers/rootlesskit/releases/download/v2.1.0/rootlesskit-aarch64.tar.gz"],
+        sha256 = "b3a9b14b66f54351a2b9f8c0917062bfeab2dc89e9e7cb15f4cf0684a4a7281f",
+    )
+
+    http_file(
+        name = "com_github_opencontainers_runc_runc-linux-amd64",
+        urls = ["https://github.com/opencontainers/runc/releases/download/v1.2.0-rc.2/runc.amd64"],
+        sha256 = "7bf38e9bc6c7a3feb5ec92e5e333015ccd0f360cdd1810a5a13f5ab81731cd62",
+        downloaded_file_path = "runc",
+        executable = True,
+    )
+    http_file(
+        name = "com_github_opencontainers_runc_runc-linux-arm64",
+        urls = ["https://github.com/opencontainers/runc/releases/download/v1.2.0-rc.2/runc.arm64"],
+        sha256 = "3741f07946c54a23ee443de84506c6e397d0d350f1f16f81ae4dd2c7d39d2651",
+        downloaded_file_path = "runc",
+        executable = True,
+    )
+
+    http_archive(
```

Comment:
would be nice to split the containerd changes into a separate PR too, if it's not too difficult